### PR TITLE
Sample unit and UI tests

### PR DESCRIPTION
**Pull request by felixlapierre**

### CODE REVIEW

File: coverage/com.example.campusguide.utils/index.html [hunk 1]
```diff
@@ -0,0 +1,125 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```
Comment (FGRCL) [inline coverage/com.example.campusguide.utils/index.html]: Do we want to keep the coverage report under version control?

File: app/src/test/java/com/example/campusguide/utils/CalculatorTest.kt [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.example.campusguide.utils
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class CalculatorTest {
+
+    @Test
```
Comment (FGRCL) [inline app/src/test/java/com/example/campusguide/utils/CalculatorTest.kt:9]: I never know when I should use hardcoded variable and when I should use test case data

**Review by FGRCL — APPROVED** [review]

File: coverage/com.example.campusguide.utils/index.html (hunk 1 above)
Comment (felixlapierre) [reply coverage/com.example.campusguide.utils/index.html]: Nope! Removing

File: app/src/test/java/com/example/campusguide/utils/CalculatorTest.kt (hunk 2 above)
Comment (felixlapierre) [reply app/src/test/java/com/example/campusguide/utils/CalculatorTest.kt:9]: Hardcoded variable is fine here, makes it clear what the inputs and expected outputs are

**Pull request merged**